File: src/structModel.java
public class structModel {
    public static void main(String[] args) {
        Vorchun vorch = new Vorchun("Ворчун", "Больной-Беглец");
        Pululka pulul = new Pululka("Пилюлька", "Больной-Беглец");
        Pulka pul = new Pulka("Пулька", "Больной");

        Hospital h = new Hospital();
        h.addPerson(vorch);
        h.addPerson(pulul);
        h.addPerson(pul);
        h.start();
    }
}

File: kotlin_fans/StructModel.kt
fun main() {
    val vorch = Vorchun("Ворчун", "Больной-Беглец")
    val pulul = Pululka("Пилюлька", "Больной-Беглец")
    val pul = Pulka("Пулька", "Больной")
    val h = Hospital()
    h.addPerson(vorch)
    h.addPerson(pulul)
    h.addPerson(pul)
    h.start()
}
